### DISCUSSION TITLE
Curso Fullstack .NET

### DISCUSSION THREAD

**Discussion by Tesshim** [balta-io/forum#642] [Tira Dúvidas] [answered]
Olá, terminei de assistir o curso Fullstack .NET, mas parece que estão faltando algumas aulas, acaba com a criação do endpoint do stripe, mas não tem a tela para utilizar a APi e em uma aula é dito que vai mostrar a criação da role premium no banco, mas não vi isso nas aulas seguintes. Esta correto ou realmente esta faltando essas informações? 
Por sinal, adorei o curso, já atuo na area o blazor wasm e mudblazor e mesmo assim o curso me agregou muito.

**Comment by atilionogueira** [comment]
Terminei de assistir ao curso Fullstack .NET e gostaria de parabenizá-los pelo excelente conteúdo.  Sou iniciante na área, mas  o curso agregou muito ao meu conhecimento e habilidades.

No entanto, percebi que parecem faltar algumas aulas no módulo de Integração com Stripe. O curso termina com a criação do endpoint do Stripe, mas não há uma tela para utilizar a API. Além disso, em uma das aulas, é mencionado que será mostrada a criação da role premium no banco, mas não encontrei essa informação nas aulas seguintes.

Gostaria de saber se essas aulas serão disponibilizadas ou se houve algum problema na disponibilização do conteúdo. Agradeço pela atenção e parabenizo novamente pelo curso.

Atenciosamente,

**Comment by andrebaltieri** [comment, marked as answer]
Bom dia, pessoal, tudo bem?

No dia 16/08 lançaremos um novo módulo do curso, um sistema completo de pedidos contemplando todas estas aulas:
👉 https://go.balta.io/jornada-stripe-dotnet

**Reply by luizaraujodev** [reply]
Show de bola! ia perguntar agora sobre isso pois vi que estava faltando a criação do StripeHandler mas vi que la no código fonte já tinha ela!
No aguardo então!

**Reply by Tesshim** [reply]
Ah, top de mais, aguardando ansiosamente pelo dia 16h
